core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregations.java

```diff
@@ -142,41 +80,33 @@ public static InternalAggregations reduce(List<InternalAggregations> aggregation
         return new InternalAggregations(reducedAggregations);
     }
 
-    /** The fields required to write this addAggregation to xcontent */
-    static class Fields {
-        public static final String AGGREGATIONS = "aggregations";
-    }
-
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
I am working on this, I was doing it but it's controversial. That's why I took it off this PR for now, I prefer doing it in small steps.